src/components/RP_GettingStarted.component

```diff
@@ -0,0 +1,82 @@
+<apex:component controller="RP_GettingStartedController">
+
+    <div class="slds-m-around_x-small">
+
+        <h1 class="slds-text-heading_large slds-p-around_medium">{!$Label.RP_GettingStarted}</h1>
+
+    </div>
+
+    <div class="slds-m-vertical_medium slds-media slds-media_responsive">
+
+        <div class="slds-p-horizontal_xx-large slds-size--1-of-1 slds-large-size--1-of-3">
+
+            <img class="slds-p-vertical_medium slds-align_absolute-center" id="trailheadImage" src="{!URLFOR($Resource.RP_Resources, 'Resources/images/Image_Trailhead.png' )}" width="150" height="60"/>
```

Comment:
The ability to modify, deprecate and manipulate the is reason enough, and I believe the only one IIRC. Should include whatever is needed in the standard resource.